controller/index.php

```diff
@@ -529,11 +530,34 @@ protected function assign_status()
 				$this->template->assign_var('ACTIVE_STATUS', $vars['NAME']);
 			}
 		}
+
+		$this->template->assign_var('SHOW_STATUS', $this->valid_type_permissions());
+	}
+
+	/**
+	 * Check whether the user has permission to filter by unapproved contributions
+	 * @return bool
+	 * @throws \Exception
+	 */
+	private function valid_type_permissions()
+	{
+		$types = \phpbb::$container->get('phpbb.titania.contribution.type.collection');
```

Comment:
Avoid using phpbb::$container if at all possible. The idea generally was to get rid of the phpbb class entirely once the DIC and extension system came into play in 3.2. The type collection is already a dependency of the index controller. Use `$this->types` instead.